library/ssl_tls.c

```diff
@@ -6033,14 +6051,14 @@ const char *mbedtls_ssl_get_alpn_protocol( const mbedtls_ssl_context *ssl )
 
 void mbedtls_ssl_conf_max_version( mbedtls_ssl_config *conf, int major, int minor )
 {
-    conf->max_major_ver = major;
-    conf->max_minor_ver = minor;
+    conf->max_major_ver = (unsigned char) major;
```

Comment:
Similar case as with mbedtls_md_get_size() - please advise if there's a way to at least indicate a possible error.